trillian.proto

```diff
@@ -64,14 +64,19 @@ enum TreeState {
   ACTIVE = 1;
 
   // Frozen trees are only able to respond to read requests, writing to a frozen
-  // tree is forbidden.
+  // tree is forbidden. Trees should not be frozen when there are entries
+  // in the queue that have not yet been integrated.
```

Comment:
maybe add a pointer to the other state: "... that have not yet been integrated (the DRAINING state covers this case)"

Reply:
Done.